error.php

```diff
@@ -0,0 +1,17 @@
+<?php
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Error</title>
+</head>
+<body>
+<h1>Something went wrong</h1>
```

Comment:
Can you provide a more meaningful message here?

Reply:
We need to provide as much feedback as possible for these guys https://imgur.com/X17puIB